vector/src/main/java/im/vector/app/features/invite/AutoAcceptInvites.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.invite
+
+import javax.inject.Inject
+
+interface AutoAcceptInvites {
+    val isEnabled: Boolean
+    val hideInvites: Boolean
```

Comment:
Default could be `= isEnabled` in the itf, no?
Also would be nice to add some Kdoc

Reply:
Yes will add some.